src/panels/usesSummaryPanel/UsesListSummaryPanel.tsx

```diff
@@ -0,0 +1,59 @@
+import { Grid } from "@material-ui/core";
+import { IUse } from "entities/IUse";
+import { IUsesGateway } from "gateways/IUsesGateway";
+import React, { FunctionComponent, useEffect, useState } from "react";
+import { UseSummaryPanel } from "./UseSummaryPanel";
+
+interface UsesListSummaryPanelProps {
+  usesGateway: IUsesGateway;
+  beaconId: string;
+}
+
+export const UsesListSummaryPanel: FunctionComponent<UsesListSummaryPanelProps> = ({
+  usesGateway,
+  beaconId,
+}: UsesListSummaryPanelProps): JSX.Element => {
+  const [uses, setUses] = useState<IUse[]>([]);
+
+  useEffect((): (() => void) => {
+    let isMounted = true;
+    const fetchUses = async (id: string) => {
+      try {
+        const uses = await usesGateway.getUses(id);
+        if (isMounted) {
+          setUses(uses);
+        }
+      } catch (error) {
+        console.error(error);
+      }
+    };
+
+    fetchUses(beaconId);
+
+    return () => {
+      isMounted = false;
+    };
+  }, [beaconId, usesGateway]);
```

Comment:
What's the purpose of `isMounted`? The component works fine without it.  Testing it locally and so far as I can tell the behaviour is identical (API calls, rendering etc.) both with and without that code?